scripts/run_driver_tests.psm1

```diff
@@ -1,4 +1,4 @@
-﻿# Copyright (c) eBPF for Windows contributors
+# Copyright (c) eBPF for Windows contributors
```

Comment:
I can't tell... what is the change in this line?

Reply:
file removed from PR.